responsibleai/responsibleai/_managers/error_analysis_manager.py

```diff
@@ -179,13 +196,17 @@ def __init__(self, model, dataset, target_column,
                                        self._feature_names,
                                        self._categorical_features)
 
-    def add(self, max_depth=3, num_leaves=31, filter_features=None):
+    def add(self, max_depth=3, num_leaves=31,
+            min_child_samples=20, filter_features=None):
```

Comment:
Can you add a unit test in model analysis to test for min_child_samples?

Reply:
done, added some basic validation logic for any trees constructed in all model analysis tests